tests/phpunit/integration/Core/Modules/Module_Sharing_SettingsTest.php

```diff
@@ -0,0 +1,111 @@
+<?php
+/**
+ * Module_Sharing_SettingsTest
+ *
+ * @package   Google\Site_Kit\Tests\Core\Modules
+ * @copyright 2022 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Tests\Core\Modules;
+
+use Google\Site_Kit\Context;
+use Google\Site_Kit\Core\Modules\Module_Sharing_Settings;
+use Google\Site_Kit\Core\Storage\Options;
+use Google\Site_Kit\Tests\Modules\SettingsTestCase;
+
+class Module_Sharing_SettingsTest extends SettingsTestCase {
+
+	/**
+	 * @inheritDoc
+	 */
```

Comment:
@aaemnnosttv why do we need this phpdoc tag in setting tests?

Reply:
@eugene-manuilov I don't think it's needed, but it seems to be used in several other places.

E.g.
- https://github.com/google/site-kit-wp/commit/cfc7a33a54d6579834ed8e4d5133d8acc786e18d#diff-cf79c1c0564dc8b2b41a66557570d042535518601c25f05c567bc500942c43e3R49
- https://github.com/google/site-kit-wp/commit/266d067efd7bd8f5dd15db5f799d59fff4c971b5#diff-3abda76dcad4f6e726d3e69df503252dc93dcb1b7334846332bd0bf1cddb002cR57

These are actually yours 😄  Could be something that someone's IDE added automatically and we didn't think about it too much. It doesn't seem to offer much benefit (at least here) so I'd be happy to remove it; not a big deal either way though.